.circle/buildenv_st2.sh

```diff
@@ -1,21 +1,27 @@
 #!/bin/bash
 set -e
+set -x
```

Comment:
Yeah, although the default output is very unfriendly - if there is an issue in this script it will just show "script exited with 1" which provides no helpful / debug info on failure.

Reply:
We really touch this part of st2-packages once in a year.

With that much verbosity what's more important like "export" statements with actual values, - they become less visible and harder to quick-read.
This noise makes it harder to see what's important in a day-to-day builds.

Alternative is to improve script error reporting for the edge cases, instead of using debug mode `-x`.

Reply:
I will remove it for now since I don't have the motivation to re-do all the error handling in that script.